acme4j-utils/src/main/java/org/shredzone/acme4j/util/CSRBuilder.java

```diff
@@ -183,6 +185,33 @@ public void addIdentifiers(Collection<Identifier> ids) {
     public void addIdentifiers(Identifier... ids) {
         Arrays.stream(ids).forEach(this::addIdentifier);
     }
+    
+    /**
+     * Sets an entry of the subject used for the CSR
+     * <p>
+     * Note that it is at the discretion of the ACME server to accept this parameter.
+     * @param attName The BCStyle attribute name
+     * @param value The value
```

Comment:
Your change will be part of the next release, so you can add ` @since 2.14` to both methods.

Reply:
done